README.rst

```diff
@@ -146,12 +146,26 @@ streams in the session to individual files (instead of a single composed file) b
   # Store this archive_id in the database
   archive_id = archive.id
 
+To add an individual stream to an archive, use the
+``opentok.add_archive_stream(archive_id, stream_id, has_audio, has_video)`` method:
+
+.. code:: python
+
+  opentok.add_archive_stream(archive.id, stream_id, has_audio=True, has_video=True)
+
+To remove a stream from an archive, use the ``opentok.remove_archive_stream()`` method:
+
+.. code:: python
+
+  opentok.remove_archive_stream(archive.id, stream_id)
+
 Composed archives (output_mode=OutputModes.composed) have an optional ``resolution`` parameter.
-If no value is supplied the opentok platform will use the default resolution "640x480".
+If no value is supplied, the archive will use the default resolution, "640x480".
```

Comment:
Composed archives also support the same resolutions as live-streaming broadcasts: "640x480" (SD landscape, the default), "1280x720" (HD landscape), "1920x1080" (FHD landscape), "480x640" (SD portrait), "720x1280" (HD portrait), or "1080x1920" (FHD portrait). See https://tokbox.com/developer/rest/#start_archive.

Reply:
Have made this change

Reply:
Have made this change